src/baseline.rs

```diff
@@ -247,29 +294,34 @@ impl BaselineLoader for RustdocBaseline {
 }
 
 pub(crate) struct PathBaseline {
-    root: PathBuf,
-    lookup: std::collections::HashMap<String, (String, PathBuf)>,
+    project_root: PathBuf,
+    lookup: std::collections::HashMap<String, Manifest>,
+    target_root: PathBuf,
 }
 
 impl PathBaseline {
-    pub(crate) fn new(root: &std::path::Path) -> anyhow::Result<Self> {
+    pub(crate) fn new(
+        // Path to a directory with the manifest or with subdirectories with the manifests.
+        project_root: &std::path::Path,
+
+        // Path to a directory where the placeholder manifest / rustdoc can be created.
+        target_root: &std::path::Path,
+    ) -> anyhow::Result<Self> {
```

Comment:
The comment content is great!

Let's use standard doc comment style for them so the text becomes visible to other tools, like rust-analyzer and editor autocomplete: https://doc.rust-lang.org/rust-by-example/meta/doc.html

Reply:
Done in 641c2f4283a6af2e8cb7597cd3ec66cc65359add.

Reply:
Nope, nevermind.

```
doc comments are not allowed here
```

Reply:
I've reverted the changes. What do you propose to do now?

Reply:
Done in https://github.com/obi1kenobi/cargo-semver-checks/pull/341/commits/4a70618e34c59fb2880356211c1d010e86f73bcd.